### PR TITLE
INTEROP-845 incorporate fuse conf changes for the tvo 5.0

### PR DESCRIPTION
**Pull request by trilio-shiwanand**

### CODE REVIEW

File: kolla-ansible/ansible/roles/triliovault/templates/fuse.conf [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# /etc/fuse.conf - Configuration file for Filesystem in Userspace (FUSE)
```
Comment (shyam-biradar) [inline kolla-ansible/ansible/roles/triliovault/templates/fuse.conf:1]: Please add .j2 extension to this template name.

File: kolla-ansible/ansible/roles/triliovault/tasks/config.yml [hunk 2]
```diff
@@ -25,6 +25,20 @@
   when:
     - inventory_hostname in groups[triliovault_datamover_group]
 
+- name: copy fuse conf file
+  become: true
+  template:
+    src: fuse.conf
+    dest: "/etc/fuse.conf"
```
Comment (shyam-biradar) [inline kolla-ansible/ansible/roles/triliovault/tasks/config.yml:32]: Permission and ownership should be like this.  42436 is nova user and group id.
      file { "/etc/triliovault-wlm/fuse.conf":
          ensure  => present,
          content => template('trilio/fuse.conf.erb'),
          owner  => '42436',
          group  => '42436',
          mode   => '0644',

**Review by shyam-biradar — CHANGES_REQUESTED** [review]

File: kolla-ansible/ansible/roles/triliovault/tasks/config.yml (hunk 2 above)
Comment (trilio-shiwanand) [reply kolla-ansible/ansible/roles/triliovault/tasks/config.yml:32]: it's resolved.

File: kolla-ansible/ansible/roles/triliovault/templates/fuse.conf.j2 [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# /etc/fuse.conf - Configuration file for Filesystem in Userspace (FUSE)
+# Set the maximum number of FUSE mounts allowed to non-root users.
+# The default is 1000.
+mount_max = 2000
+# Allow non-root users to specify the allow_other or allow_root mount options.
+user_allow_other
```
Comment (shyam-biradar) [inline kolla-ansible/ansible/roles/triliovault/templates/fuse.conf.j2:6]: We need to add a empty line at the end of this file. Otherwise it throws error.

Comment (trilio-shiwanand) [reply kolla-ansible/ansible/roles/triliovault/templates/fuse.conf.j2:6]: Okay, will add it.

**Review by shyam-biradar — APPROVED** [review]

**Pull request merged**